### PR TITLE
Select component that works for a basic dropdown

### PR DESCRIPTION
**Pull request by agclever**
JIRA: https://clever.atlassian.net/browse/DIS2-179

Creates a new select component that works as a basic dropdown for now. Wraps react-select with styling based on mocks at https://projects.invisionapp.com/share/TG89776AU#/screens/180681819.

Wraps react-select and applies custom styling.

**Screenshots**

Focused on an option with none selected:
screenshot 2016-08-24 16 24 10

Focused with an option selected:
screenshot 2016-08-24 16 24 18

### CODE REVIEW

**Comment by agclever** [comment]
cc: @CarlNelson @joshatclever

**Comment by CarlNelson** [comment]
Change display: none on label--hidden to opacity: 0 to allow for screen readers to read it. Use a transition to fade it in.

**Comment by agclever** [comment]
Changed to use opacity in 72aef722c55f06aade155401164d626f. Did not use transition because the placeholder does not have transitions and there isn't a straightforward way for us to make sync those up

**Comment by ipam73** [comment]
@ag could you document the new component on the readme?

**Comment by agclever** [comment]
Updated README in ceb6b37

File: src/Select/Select.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          placeholder={placeholder}
+          searchable={false}
+          value={value}
+        />
+      </div>
+    </div>
+  );
+}
+
+Select.cssClass = {
```
Comment (ipam73) [inline src/Select/Select.jsx]: nice way to simplify css classes!

File: src/Select/Select.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  CONTAINER: "Select--container",
+  LABEL: "Select--label",
+  LABEL_CONTAINER: "Select--labelContainer",
+  LABEL_HIDDEN: "Select--labelHidden",
+  REACT_SELECT: "Select--ReactSelect",
+};
+
+const selectValue = React.PropTypes.shape({
+  label: React.PropTypes.string,
```
Comment (ipam73) [inline src/Select/Select.jsx]: are these required for a select value?  or are they both optional?

File: src/Select/Select.less [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      }
+      .Select-value {
+        font-size: 15px;
+        line-height: 40px;
+        min-height: 40px;
+        padding-top: 10px;
+      }
+    }
+
+    // overrides styles from the exact same selector in react-select's CSS
```
Comment (ipam73) [inline src/Select/Select.less]: I'm not too familiar with best practices overriding other selections (if there's any) -- would probably defer on @CarlNelson on this

**Comment by ipam73** [comment]
@agclever this looks good to me!  I'm deferring comments on style/design to @joshatclever and @CarlNelson

File: src/Select/Select.jsx (hunk 2 above)
Comment (agclever) [inline src/Select/Select.jsx]: Good question. Made them both required in https://github.com/Clever/components/pull/46/commits/97885be5613578bf50b4460a703632e940c2909a

File: src/Select/Select.less (hunk 3 above)
Comment (CarlNelson) [inline src/Select/Select.less]: Font size should be in them, Select value should be 1rem (16px)

Comment (agclever) [inline src/Select/Select.less]: Changed to use rem in 5765fe5521102e9ee23b90cdf671821082965b2f.

File: src/Select/Select.jsx (hunk 1 above)
Comment (agclever) [inline src/Select/Select.jsx]: Borrowing from @kofi-clever's code

**Pull request merged**